<?xml version="1.0"?>
<launch>
  <param name="robot_description" command="xacro --check-order '$(find xiaomi_bridge)/urdf/xiaomi_gen1.urdf'" />
  <node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher">
    <param name="publish_frequency" type="double" value="5.0" />
  </node>

  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
    <param name="use_gui" value="false"/>
  </node>
  <node pkg="xiaomi_bridge" type="xiaomi_bridge_node" name="xiaomi_bridge_node">
    <param name="vacuum_ip" type="str" value="192.168.0.20" />
  </node>
</launch>
